pallets/elections/src/traits.rs

```diff
@@ -86,3 +86,21 @@ where
         pallet_staking::ErasStakers::<T>::iter_key_prefix(era).collect()
     }
 }
+
+pub trait ValidatorExtractor {
+    type AccountId;
+
+    /// Removes given validator from pallet's staking valiators list
```

Comment:
typo, should be -> Removes given validator from pallet's staking validator list

Reply:
Fixed.